EIPS/eip-2981.md

```diff
@@ -16,21 +16,23 @@ A standardized way to retrieve royalty payment information for ERC-721 tokens to
 
-This standard extends the [ERC-721 specification](./eip-721.md) to enable setting a royalty amount paid to the NFT creator or rights holder every time an NFT is sold and re-sold. If a marketplace chooses *not* to implement this EIP, then obviously no funds are paid for secondary sales. But as most NFT marketplaces have developed some unique royalty system themselves - and all of them are singular and only work within their own contracts - there should be an accepted standard for royalty payment information (if the creator chooses to set royalties on their NFTs). We believe the NFT marketplace ecosystem will voluntarily implement royalty payments to provide ongoing funding for artists and other creators, and NFT buyers will assess the royalty payment as a factor when making NFT purchasing decisions.
+This standard is intended as a complement to the [ERC-721 specification](./eip-721.md) and [ERC-1155 specification](./eip-1155.md), allowing the NFT creator or rights holder to signal the royalty amount to be paid every time an NFT is sold and re-sold. If a marketplace chooses *not* to implement this EIP, then obviously no funds are paid for secondary sales. But as most NFT marketplaces have developed some unique royalty system themselves - and all of them are singular and only work within their own contracts - there should be an accepted standard for royalty payment information (if the creator chooses to set royalties on their NFTs). We believe the NFT marketplace ecosystem will voluntarily implement royalty payments to provide ongoing funding for artists and other creators, and NFT buyers will assess the royalty payment as a factor when making NFT purchasing decisions.
 
 Without an agreed royalty payment standard, the NFT ecosystem will lack an effective means to collect royalties across all marketplaces and artists and other creators will not receive ongoing funding. This will hamper the growth and adoption of NFTs and demotivate artists and other NFT creators from minting new and innovative tokens.
 
 *"Yes we have royalties, but if your NFT is sold on another marketplace, we cannot provide royalties" ... "But can't I sell my NFT anywhere with a click of my wallet?" ... "Yes... but we don't have a standard for royalties so you'll lose out."*
 
 This EIP fixes this issue, enabling all NFT marketplaces to unify on a single royalty payment information standard and benefiting the entire NFT ecosystem.
 
+While, ERC721 NFTs are the primary use case at this time, we envision that the functionality can be applied to all extensions to ERC721 and other contract interfaces that wish to derive royalties in this way. 
```

Comment:
```suggestion
While, ERC-721 NFTs are the primary use case at this time, we envision that the functionality can be applied to all extensions to ERC721 and other contract interfaces that wish to derive royalties in this way. 
```

Reply:
You'd also want to hyphenate the second instance in that same paragraph.